While executing file `Games/1TileMap/main.py` an error occurred (<module> refers to the game file):

Traceback (most recent call last):
  File "/mnt/c/Users/Mark/Thumby/TinyCircuits-Tiny-Game-Engine/filesystem/launcher.py", line 307, in <module>
  File "main.py", line 27, in <module>
  File "/mnt/c/Users/Mark/Thumby/TinyCircuits-Tiny-Game-Engine/filesystem/Games/1TileMap/tilemapper.py", line 46, in __init__
  File "/mnt/c/Users/Mark/Thumby/TinyCircuits-Tiny-Game-Engine/filesystem/Games/1TileMap/tilemapper.py", line 54, in render_tiles
  File "/mnt/c/Users/Mark/Thumby/TinyCircuits-Tiny-Game-Engine/filesystem/Games/1TileMap/tilemapper.py", line 70, in render_tile
AttributeError: 'str' object has no attribute 'texture'
